main.yml

```diff
@@ -379,6 +379,22 @@ insights:
       - id: ros
   top_level: true
 
+integrations:
+  title: Integrations
+  channel: '#cloudservices-notifications'
+  api:
+    versions:
+      - v1
+  frontend:
+    app_base: notifications
+    module:
+      appName: integrations
+      scope: notifications
+      module: ./RootApp
+      manifest: /apps/notifications/fed-mods.json
```

Comment:
Should I remove this one on stable? `validate-yaml` is complaining

Reply:
We didn't updated the yaml validation on stable branch as of yet - this PR should solve your problem https://github.com/RedHatInsights/cloud-services-config/pull/522